tests/installation/addon_products_via_SCC_yast2.pm

```diff
@@ -0,0 +1,47 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: add addon to SLES via SCC
+# Maintainer: Jozef Pupava <jpupava@suse.com>
+
+use base "y2logsstep";
+use strict;
+use testapi;
+use registration 'fill_in_registration_data';
+
+sub run() {
+    x11_start_program('xterm');
```

Comment:
It is in installation same as e.g. [sles+rt](https://openqa.suse.de/tests/715106)